# 1. Simulator parameters
use_sim_time: true
clockscale: 1.0                         # only 1.0 is supported yet

# 2. Vehicle initial geodetic position
# Kazanexpress point:
lat_ref : 55.7503992494
lon_ref : 48.7481202714
alt_ref : -6.5

# 3. Vehicle initial local position
# X, Y, Z, roll, pitch, yaw, weight
# N - [0,0,0,1], E - [0,0,0.7,0.7], S - [0,0,1,0], W - [0,0,0.7,-0.7]
init_pose: [0.00, 0.00, 0.0,
            0.00, 0.00, +0.00, +1.00]

# 4. Additional sensors set
esc_status: true
ice_status: true
fuel_tank_status: true
battery_status: true

# Environment parameters
wind_ned: [5.0, 0.0, 0.0]
wind_variance: 0.0
gust_ned: [0.0, 0.0, 0.0]
gust_variance: 0.0
